9c-internal/all-in-one/values.yaml

```diff
@@ -0,0 +1,145 @@
+# Default values for 9c-internal cluster.
+# This is a YAML-formatted file.
+
+
+clusterName: "9c-internal-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    tag: "git-7c0815d08cb5d94700918ca4598ae2634c26ac1e"
```

Comment:
as you can read from chart template, global image tag could be overwritten by the tag of each component(validator, remote-headless, etc.). so if you want to use global image tag, remove image tag from those using it